src/signature-help.jl

```diff
@@ -375,10 +360,8 @@ function cursor_call(ps::JS.ParseStream, st0::JL.SyntaxTree, b::Int)
     end
 
     # `i` is nothing.  Eat preceding whitespace and check again.
-    pnb_line = prev_nontrivia_byte(ps, b; pass_newlines=false)
-    pnb = prev_nontrivia_byte(ps, b; pass_newlines=true)
-
-    (isnothing(pnb) || pnb === b) && return nothing
+    pnb = prev_nontrivia_byte(ps, b-1; pass_newlines=true)
```

Comment:
`-1` because now we have `1 ≤ b ≤ JS.last_byte(ps) || return nothing` check in `prev_nontrivia`, but I believe giving `b-1` here is more correct?